### ISSUE TITLE
Basic captcha is likely not EAA compliant

### ISSUE THREAD

**Issue by makuhlmann** [shopware/shopware#10870]
### Shopware Version

6.6.10.5 / 6.7.0.0

### Affected area / extension

Platform(Default)

### Actual behaviour

The basic captcha is likely not compliant with the new EAA rules. Not only does the image not contain alt text, nor does the button have a discernible text explaining what it does, but the captcha cannot be solved by people without eyesight or relying on screen readers either.

![Image](https://github.com/user-attachments/assets/4efac50a-a919-4433-9cff-73624f384f47)

Fixing this would likely require a rewrite of the feature, but a warning could already help, when shop owners try to enable this captcha type.

### Expected behaviour

The basic captcha would be EAA compliant.

### How to reproduce

- Set up a new 6.7 shop and enable basic captcha
- Analyse the code of the registration page using F12, observe no alt texts for the captcha image, nor the button next to the image
- You can also scan the site using a scanner: https://www.accessibilitychecker.org/

### Definition of Done

- [ ] An automated test covers the fix.
- [ ] Integration/E2E testing in staging is done.
- [ ] Changelog markdown file created or updated in /changelog/_unreleased directory.
- [ ] Developer documentation written/updated.
- [ ] End user and/or developer documentation written or updated with clarification.
- [ ] Regression test created to ensure bug does not reappear, and automated regression tests pass.
- [ ] Code changes resolve the reported bug without introducing new issues.

**Comment by keulinho** [comment]
Backport is merged so I'll close this again, so this will land in the next 6.6.10.x release as well
